docker/service/src/test/java/uk/nhs/utils/InvestigationMapperArgumentsProvider.java

```diff
@@ -0,0 +1,30 @@
+package uk.nhs.utils;
+
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+import org.springframework.core.io.support.ResourcePatternResolver;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class InvestigationMapperArgumentsProvider implements ArgumentsProvider {
+
+    @SneakyThrows
+    public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
+        return Arrays.stream(getResources())
```

Comment:
the next one of these we should unify these - not a big one for now but getting to the point where we are repeating lots of code!